lib/loader/index.js

```diff
@@ -65,6 +65,32 @@ function instantiate(module, imports) {
     return parts.join("") + String.fromCharCode.apply(String, U16.subarray(dataOffset, dataOffset + dataRemain));
   }
 
+  /** Gets an array from the module's memory by its pointer and its type. */
+  function getArray(ptr, type, asTippedArray) {
+    checkMem();
+    const ptr0 = ptr >>> 2;
+    const dataStart = (U32[ptr0] >>> 2) + 2; // 0 -> size; 1 -> static 0
+    const len = U32[ptr0 + 1];
+    const transform = asTippedArray ? (x) => x : (x) => Array.from(x);
```

Comment:
Should this be `typedArray`? And is it actually necessary? `String` would naturally have to return a normal array of strings because it can't be a typed one, but everything else should be ok when sliced from the underlying views as a typed one.

Reply:
LGTM. I removed this.